Tests/Snapper_test.py

```diff
@@ -2,6 +2,11 @@
 from Snapper.Snapper import Snapper
 
 
+class TestManager:
+    def on_data(self, snapshot):
+        self.snapshot = snapshot
```

Comment:
My gut tells me there should be a way to get around doing this. If you create a TestManager class, you aren't really testing the Manager anymore. Should be able to mock out routes if you need to.